templates/rhsm-katello-reconfigure.erb

```diff
@@ -95,6 +95,11 @@ fi
   service goferd status >/dev/null && \
   service goferd restart >/dev/null 2&>1
 
+
+if [ -d /etc/rhsm/facts/ ]; then
+  echo "{\"network.hostname-override\":\"`hostname -f`\"}" > /etc/rhsm/facts/katello.facts
```

Comment:
Make sense. Last question, override makes some sense, however `hostname -f` is a valid value and fact. Would it be worth calling it `hostname-full` ?